src/module/items/base-sheet.ts

```diff
@@ -1,3 +1,8 @@
+interface EditorOptions {
+  target: Record<string, unknown>;
+  height: number;
+  save_onsavecallback: () => Promise<void>;
+}
```

Comment:
Once again, off topic for the PR's stated goal

Reply:
Done as part of #32.